src/python/pants/backend/project_info/tasks/idea_plugin_gen.py

```diff
@@ -167,7 +162,10 @@ def _generate_to_tempfile(self, generator):
       generator.write(output)
       return output.name
 
-  def execute(self):
+  def console_output(self, _targets):
```

Comment:
I'm not sure how to handle this `targets` argument to `console_output`. Originally, the task was using `self.context.target_roots` and `self.context.options.target_specs`. Both of these things are not the same as the `self.context.targets` that `ConsoleTask` passes on in its call to `console_output()`.

Is it fine as is or is there a way to make use of the `targets` param?

Reply:
Fine as is.